src/main/java/uk/ac/ebi/eva/commons/mongodb/filter/VariantEntityRepositoryFilter.java

```diff
@@ -37,27 +36,25 @@
     public final static String STUDY_ID_FIELD = FILES_FIELD + ".sid";
     public final static String FILE_ID_FIELD = FILES_FIELD + ".fid";
 
-    public final static String CONSEQUENCE_TYPE_FIELD = ANNOTATION_FIELD + ".ct";
-    public final static String POLYPHEN_FIELD = CONSEQUENCE_TYPE_FIELD + ".polyphen.sc";
-    public final static String SIFT_FIELD = CONSEQUENCE_TYPE_FIELD + ".sift.sc";
-    public final static String CONSEQUENCE_TYPE_SO_FIELD = CONSEQUENCE_TYPE_FIELD + ".so";
+    public final static String POLYPHEN_FIELD = "annot.polyphen";
+    public final static String SIFT_FIELD = "annot.sift";
+    public final static String CONSEQUENCE_TYPE_SO_FIELD = "annot.so";
 
     public final static String XREFS_FIELD = ANNOTATION_FIELD + ".xrefs";
     public final static String XREFS_ID_FIELD = XREFS_FIELD + ".id";
 
     private final String field;
     private final T value;
-    private final VariantEntityRepository.RelationalOperator operator;
+    private final RelationalOperator operator;
 
-    public VariantEntityRepositoryFilter(String field, T value, VariantEntityRepository.RelationalOperator operator) {
+    public VariantEntityRepositoryFilter(String field, T value, RelationalOperator operator) {
```

Comment:
Shouldn't this class hierarchy names be derived from `VariantRepositoryFilter` instead of `VariantEntityRepositoryFilter`?